charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -328,6 +338,18 @@ node:
     runAsUser: 0
     runAsGroup: 0
     fsGroup: 0
+  # Add additional volume mounts on the node pods with node.volumes and node.volumeMounts
+  volumes: [ ]
```

Comment:
Let's do `[  ]` -> `[]` for consistency, similar to https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/280ac42f577409c1976b88cb2ebd0d9c3d161725/charts/aws-ebs-csi-driver/values.yaml#L267

Same request at 348